src/components/MultiSelectCombobox/ComboboxItem.tsx

```diff
@@ -0,0 +1,20 @@
+import type { MouseEventHandler } from 'react';
+import React from 'react';
+import DropdownItem from '../Dropdown/DropdownItem';
+
+export interface ComboboxItemProps {
+  children: string;
+  onClick: MouseEventHandler;
+}
+
+const ComboboxItem = ({ children, onClick }: ComboboxItemProps) => (
+  <DropdownItem
+    onClick={onClick}
+    className="overflow-hidden text-truncate"
+    title={children as string}
```

Comment:
Can we remove the type casting here?

Reply:
Yes! I just saw that one myself. Now that `children` is `string`, that can go. Much better.